gstd/src/common/handlers.rs

```diff
@@ -25,27 +25,26 @@
 //! debug and non-debug mode, for programs built in `wasm32` architecture.
 //! For `debug` mode it provides more extensive logging.
 
-#[cfg(target_arch = "wasm32")]
-use {crate::ext, core::alloc::Layout, core::panic::PanicInfo};
-
+#[cfg(feature = "oom-handler")]
 #[cfg(target_arch = "wasm32")]
 #[alloc_error_handler]
-pub fn oom(_: Layout) -> ! {
-    ext::oom_panic()
+pub fn oom(_: core::alloc::Layout) -> ! {
+    crate::ext::oom_panic()
 }
 
 #[cfg(not(any(feature = "debug", debug_assertions)))]
 #[cfg(target_arch = "wasm32")]
 #[panic_handler]
 pub fn panic(_: &PanicInfo) -> ! {
-    ext::panic("no info")
+    crate::ext::panic("no info")
 }
 
+#[cfg(feature = "panic-message")]
 #[cfg(any(feature = "debug", debug_assertions))]
```

Comment:
It was a long time I was thinking of removal here cfg(debug_assertions) and the time has come, isn't it?

Reply:
Done in 24351be